libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/routeline/RouteLineComponent.kt

```diff
@@ -42,11 +42,19 @@ internal class RouteLineComponent(
     }
 
     private val routesObserver = RoutesObserver { result ->
-        val routeLines = result.routes.map { RouteLine(it, null) }
+        val routeLines = result.navigationRoutes
         jobControl.scope.launch {
-            routeLineApi.setRoutes(routeLines).let { routeDrawData ->
+            if (routeLines.isNotEmpty()) {
```

Comment:
This is incorrect. The empty collection should be passed into `routeLineApi.setNavigationRoutes()`.

Reply:
Removed.